### PR TITLE
OCPBUGS-19873: Created Cri-O default capabilities section

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.13+

Issue: [OCPBUG 19873](https://issues.redhat.com/browse/OCPBUGS-19873)

Link to docs preview: [Doc Preview](https://68973--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/machine-configuration-tasks#create-crio-default-capabilities_post-install-machine-configuration-tasks)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by sergiordlr** [comment]
LGTM

**Comment by umohnani8** [comment]
LGTM

File: modules/create-crio-default-capabilities.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="create-crio-default-capabilities_{context}"]
+= Creating a drop-in file for CRI-O's default capabilities
+
+You can change some of the settings associated with the {product-title} CRI-O runtime for the nodes associated with a specific machine config pool (MCP). Using a `ContainerRuntimeConfigController` (CRCC) custom resource (CR), you set the configuration values and add a label to match the MCP. The MCO then rebuilds the `crio.conf` and `default.conf` configuration files on the associated nodes with the updated values.
```
Comment (rioliu-rh) [inline modules/create-crio-default-capabilities.adoc:9]: I think there is no CR with name `ContainerRuntimeConfigController`

**Comment by rioliu-rh** [comment]
/retitle OCPBUG-19873: Created Cri-O default capabilities section

**Comment by rioliu-rh** [comment]
/retitle OCPBUGS-19873: Created Cri-O default capabilities section

File: modules/create-crio-default-capabilities.adoc (hunk 1 above)
Comment (rioliu-rh) [reply modules/create-crio-default-capabilities.adoc:9]: this sentence makes me a little confuse. maybe we don't have to mention the controller to the end user. they just need to know which CR can be used to change the config

File: modules/create-crio-default-capabilities.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+.Example output
+[source,terminal]
+----
+NAME         AGE
+ctr-overlay  15m
+ctr-level    5m45s
+----
+
+.Example showing multiple `containerruntime` machine configs
```
Comment (rioliu-rh) [inline modules/create-crio-default-capabilities.adoc:36]: use `machine config` here will make the user confused. it is not the CR managed by MCO, actually the cmd output is showing the capabilities.

Comment (subhtk) [reply modules/create-crio-default-capabilities.adoc:36]: @rioliu-rh Any suggestion regarding the example title?

Comment (rioliu-rh) [reply modules/create-crio-default-capabilities.adoc:36]: MachineConfig is CR managed by MCO, below command is used to show container runtime related system config on cluster node. 
my suggestion is: `showing multiple containerruntime related system configs`

**Pull request closed without merging**